src/components/ProgressRow/ProgressRow.jsx

```diff
@@ -0,0 +1,108 @@
+import React, {useEffect, useState} from "react";
+import './ProgressRow.css';
+
+const ProgressRow = ({taskList}) => {
+
+    const [boards, setBoards] = useState([]);
+
+    useEffect(() => {
+        setBoards([
```

Comment:
we don't need this useEffect, move boards to separate const with useMemo
const defaultBoards = useMemo(() => (
[
            {id: 1, title: "TO DO", items: taskList},
            {id: 2, title: "READY FOR DEV", items: []},
            {id: 3, title: "IN PROGRESS", items: []},
            {id: 4, title: "DONE", items: []},
        ]
), [taskList]);
const [boards, setBoards] = useState(defaultBoards);